packages/eas-cli/src/metadata/config.ts

```diff
@@ -12,6 +18,81 @@ export interface MetadataConfig {
   apple?: AppleMetadata;
 }
 
+/**
+ * Resolve the dynamic config from the user.
+ * It supports methods, async methods, or objects (json).
+ */
+async function resolveDynamicConfigAsync(configFile: string): Promise<unknown> {
+  const userConfigContent = await import(configFile);
```

Comment:
I think this was the ESM-styled dynamic imports. I think we can use them bc it's transpiled.